xapian-bindings/python3/pythonletortest.py

```diff
@@ -98,6 +99,7 @@ def test_preparetrainingfile():
     import filecmp
     expect(filecmp.cmp(training_data, "training_output1.txt"), True)
     os.remove("training_output1.txt")
+    shutil.rmtree(dbpath)
```

Comment:
Presumably you want to clear up `dbpath` no matter what? In which case you should use `try…finally`, which ensures the cleanup happens even with exceptions. (So probably run it after setting up the database, or ensure the `rmtree()` won't fail if the directory is missing.)

Reply:
fixed in  a fixup commit.